From 5d69c74659e215a755b9411e40ac86287cd7d543 Mon Sep 17 00:00:00 2001
From: Antoine Belvire <antoine.belvire@opensuse.org>
Date: Sat, 11 Apr 2020 19:38:08 +0200
Subject: [PATCH] Remove version tag from desktop file

The version entry corresponds to the Desktop Entry Specification the desktop
file conforms with - not the application version. Let's remove this entry as
it's confusing and optional anyway.
---
 gui/resources/solarus-launcher.desktop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gui/resources/solarus-launcher.desktop b/gui/resources/solarus-launcher.desktop
index df6cfd54b..2be9d733e 100644
--- a/gui/resources/solarus-launcher.desktop
+++ b/gui/resources/solarus-launcher.desktop
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Version=1.6.2
 Type=Application
 Name=Solarus Launcher
 GenericName[es]=Lanzador de videojuegos Solarus
-- 
2.26.0

